backend/tests/dining/test_views.py

```diff
@@ -93,6 +138,16 @@ def test_get_date(self):
         response = self.client.get("/dining/menus/2022-10-04/")
         self.try_structure(response.json())
 
+    @mock.patch("requests.request", mock_request_raise_error)
+    def test_skip_vanue(self):
+        Venue.objects.all().delete()
+        Venue.objects.create(venue_id=747, name="Skip", image_url="URL")
+        wrapper = DiningAPIWrapper()
```

Comment:
Hm, I'm wondering if there is a way to assert an Error in Python Testcase (similar to 121 Java testing)?

Reply:
https://stackoverflow.com/questions/4319825/python-unittest-opposite-of-assertraises
:(